requirements.txt

```diff
@@ -75,6 +75,7 @@ typing-extensions==3.7.4.3
     #   yarl
 yarl==1.6.3
     # via aiohttp
+youtube_dl==2021.5.16
```

Comment:
`requirements.txt` ne doit pas être édité à la main (c'est écrit en haut du fichier). Il faut modifier `requirements.in` (de préférence sans pin la version sans raison) et puis le compiler avec `pip-compile`